ext/SparseMatricesCSRExt.jl

```diff
@@ -0,0 +1,26 @@
+module SparseMatricesCSRExt
+
+using CUDA
+import CUDA: CUSPARSE, CuVector
+using SparseMatricesCSR
+import SparseMatricesCSR: SparseMatrixCSR
+import Adapt
+
+CUSPARSE.CuSparseMatrixCSR{T}(Mat::SparseMatrixCSR) where {T} =
```

Comment:
What if I want to construct a `CuSparseMatrixCSC` from `SparseMatrixCSR`? Should more c-tors/conversions be added here, so that a user doesn't have to convert twice where a more direct way is available?